clusterloader2/README.md

```diff
@@ -1,21 +1,79 @@
 # ClusterLoader
 
-### Running ClusterLoader
+## Running ClusterLoader
 
 To run ClusterLoader type:
 ```
-go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.json
+go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.yaml
 ```
 Flags kubeconfig and testconfig are necessary.
 
-#### Flags
+### Flags
 
+#### Required
+
+These flags are required for any test to be run.
  - kubeconfig - path to the kubeconfig file.
- - testconfig - path to the test config file.
+ - testconfig - path to the test config file. This flag can be used multiple times
+if more than one test should be run.
 
-### Vendor
+#### Optional
 
-Vendor is created using [govendor].
+ - nodes - number of nodes in the cluster.
+If not provided, test will assign the number of available schedulable cluster nodes.
+ - report-dir - path to directory, where summaries files should be stored.
+If not specified, summaries are printed to standard log.
+
+## Tests
+
+### Test definition
+
+Test definition is an instantiation of this [api] (in json or yaml).
+The motivation and description of the API can be find in [design doc].
+Definitions of test as well as definitions of individual objects support templating.
+Templates for test definition come with one predefined value - ```{{.Nodes}}```, which represents the number of schedulable nodes in the cluster. \
+Example of a test definition can be found here: [load test].
+
+### Object template
 
+Object templates are similar to standard kubernetes object templates. The only difference is that the templating has been added.
```

Comment:
There is concept of "object template" per-se in k8s.
So I would say something like:

Object templates is similar to standard kubernetes object definition with the only difference being templating mechanism.